app/main/views/g_cloud.py

```diff
@@ -70,6 +70,8 @@ def index_g_cloud():
 
 @direct_award_public.route("/<string:framework_family>/start", methods=("GET",))
 def pre_project_task_list(framework_family):
+    if framework_family != "g-cloud":
+        abort(404)
```

Comment:
Can this view use `get_latest_live_framework_or_404` like the ones below?

Reply:
I thought so initially, but realizing it was hard coded to `"g-cloud"` (like, 3 lines down) I thought it best to stick with that.

Reply:
Yeah, I guess we don't want DOS direct awards sneaking in at the start.